docs/release_notes/v0.3.0.md

```diff
@@ -0,0 +1,77 @@
+# Dapr 0.3.0
+ 
+We're happy to announce the release of Dapr 0.3.0.
+ 
+This release focused on improvements to our end-to-end tests infrastructure, adding a few new components, improving CLI error messages and stability, and resolving bugs on and improving the Dotnet SDK.
+
+Visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page to get started with Dapr.
+
+## Upgrading to Dapr 0.3.0
+
+To get the latest bits, follow the instructions [here](https://github.com/dapr/cli#installing-dapr-cli) to download the latest and greatest CLI release and binaries.
+
+Once you have the Dapr CLI in your path, run `dapr init`.
+Wait for the update to finish, and you're good to go!
+
+## Kubernetes
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, first download the latest CLI release as outlined above.
+
+Once you have the CLI, run:
+
+```
+dapr uninstall --kubernetes
+dapr init --kubernetes
+```
+
+If installed Dapr previously via Helm, run the following command to upgrade:
+
+```
+helm upgrade dapr dapr/dapr --set global.tag=0.3.0
+```
+
+*Note: If you installed with a different chart name, use `helm upgrade <your-chart-name>`*
+
+## Dapr Runtime
+ 
+**Added** Couchbase as a State Store
```

Comment:
Service discovery componentization was in this release.

Reply:
Right! I wonder why I didn't see that as part of the project

Reply:
Fixed

Reply:
Came in right at the end of 0.2 but wasn't part of 0.2.  Probably the card got shuffled.